data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/util/KafkaSourceSecurityConfigurer.java

```diff
@@ -214,17 +212,15 @@ public static String getBootStrapServersForMsk(final AwsIamAuthConfig awsIamAuth
             retryable = false;
             try {
                 result = kafkaClient.getBootstrapBrokers(request);
-            } catch (InternalServerErrorException | ConflictException | ForbiddenException | UnauthorizedException | StsException e) {
+            } catch (KafkaException | StsException e) {
                 LOG.debug("Failed to get bootstrap server information from MSK. Retrying...", e);
```

Comment:
should this be LOG.info? also make it explicit like will retry with exponential backoff or after so many seconds. Do we need to log entire backtrace? just e.message() may be enough?